website/content/docs/configuration/client.mdx

```diff
@@ -217,27 +217,35 @@ chroot as doing so would cause infinite recursion.
   files on the client host via the `file` function. By default, templates can
   access files only within the [task working directory].
 
-- `max_stale` `(string: "")` - # This is the maximum interval to allow "stale"
+- `max_stale` `(string: "87600h")` - This is the maximum interval to allow "stale"
   data. By default, only the Consul leader will respond to queries. Requests to
```

Comment:
> By default, only the Consul leader will respond to queries.

I don't think this is true anymore right? Looks like 87.6k hours is the new default.

Reply:
Nice catch!